guides/v15-mdc-migration.md

```diff
@@ -491,47 +498,47 @@ TODO(mmalerba): link to density docs once they exist.
   is the same width as the form-field
 
 ### Slide Toggle
+<!-- TODO(zarend): review -->
 
-* Accessibility: MDC-based version uses `<button role="switch">` to represent the toggle rather than
-  `<input type="checkbox">`
+* To improve accessibility, the MDC-based version uses `<button role="switch">` to represent the
+  toggle rather than `<input type="checkbox">`.
 
-* Accessibility: The touch target is much larger than the slide toggle; to match the legacy size,
-  you can provide a `-5` density to the slide toggle’s theme mixin:
+* The touch target is much larger and more accessible. Be sure to allow enough space in your
+  layout so that the touch target does not overlap other components. If you are not concerned with
+  accessibility you can match the previous size by using density -5 for the slide-toggle.
 
   ```scss
   @use '@angular/material' as mat;
-
-  @include mat.slide-toggle-theme(
-    map-merge($theme, (density: -5))
-  );
+  @include mat.slide-toggle-density(-5px);
   ```
 
 * The label is closer to the enabled toggle
 
 ### Slider
+<!-- TODO(wagnermaciel): review -->
 
-* Accessibility: sliders now work with mobile device screen readers.
+* Sliders now work with mobile device screen readers.
 
 * To accommodate range sliders, the implementation has changed from the `<mat-slider>` element being
   the form control to the `<mat-slider>` element containing 1-2 `<input>` elements (the slider
   "thumbs") that act as the form control(s). The value, associated events (`input`, `change`), and
   labels (`aria-label`) now live on the `<input>` elements instead.
 
 * Vertical sliders and inverted sliders are no longer supported, as they are no longer part of the
-  Material Design spec. TODO(jelbourn): should we add a note that vertical sliders are likely to
-  return in a future version?
+  Material Design spec.
+  <!-- TODO(jelbourn): should we add a note that vertical sliders are likely to return in a future version? -->
```

Comment:
@jelbourn Thoughts about whether we should mention this now?

Reply:
I'm going to say "no" because I don't want to be making promises about what MDC team will/won't deliver.